src/mscorlib/shared/System/Span.NonGeneric.cs

```diff
@@ -28,13 +28,45 @@ public static class Span
         public static ReadOnlyMemory<char> AsReadOnlyMemory(this string text)
         {
             if (text == null)
-            {
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.text);
-            }
 
             return new ReadOnlyMemory<char>(text, 0, text.Length);
         }
 
+        /// <summary>Creates a new <see cref="ReadOnlyMemory{T}"/> over the portion of the target string.</summary>
+        /// <param name="text">The target string.</param>
+        /// <exception cref="System.ArgumentNullException">Thrown when <paramref name="text"/> is a null reference (Nothing in Visual Basic).</exception>
```

Comment:
Is it necessary to specify "Nothing in Visual Basic". If so, should we add that to other comments as well (for example: https://github.com/dotnet/corefx/blob/master/src/System.Memory/src/System/MemoryExtensions.Portable.cs#L63). Is there a way to indicate that `Nothing` is a keyword?

Reply:
The phrase appears in various XML docs (probably all copy-pasted from one instance as these were.) I don't really have a dog in this either way - I'd rather that this PR not be used as the place to debate that issue, though.

Reply:
Fair enough.